gauge-browser-agent-otel/services/sealightsService.js

```diff
@@ -0,0 +1,45 @@
+const axios = require("axios");
+const jwtDecode = require("jwt-decode");
+const fs = require("fs");
+
+const apiToken = fs.readFileSync(`${__dirname}/../sltoken.txt`, "utf-8");
+const bsid = fs.readFileSync(`${__dirname}/../buildSessionId`, "utf-8");
+
+const decoded = jwtDecode(apiToken); // Agent Token
+const baseUrl = decoded["x-sl-server"]; // Base url of the backend
+
+const http = axios.create({
```

Comment:
Better use a specific name, for xample testSessionsV1Instance  and testSessionsV2Instance